app/controllers/sign_in_security_check_failed_controller.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class SignInSecurityCheckFailedController < ApplicationController
+  def show
+    analytics.security_check_failed_visited
```

Comment:
Same note as before about disambiguating name, can we align the event name to the new controller name?